vagrant/vagrant.yml

```diff
@@ -1,15 +1,15 @@
 ---
-- hosts: "*fedora*"
+- hosts: "*fedora*, subman-devel"
```

Comment:
Unnecessary change, and might actually cause issues as written. Just don't change this line.

Reply:
OK